src/bin/cargo/commands/clean.rs

```diff
@@ -17,29 +21,243 @@ pub fn cli() -> Command {
         .arg(
             flag(
                 "dry-run",
-                "Display what would be deleted without deleting anything",
+                "Display what would be deleted without deleting anything (unstable)",
             )
             .short('n'),
         )
+
+        // NOTE: Not all of these options may get stabilized. Some of them are
+        // very low-level details, and may not be something typical users need.
+        .arg(
+            optional_opt(
+                "gc",
+                "Delete old and unused files (unstable) (comma separated): all, download, target, shared-target",
```

Comment:
Clap will list out the values for you because you used `PossibleValueParser`

(this is mostly a nit but don't want to forget this as we move forward)